lan867x/examples/lan867x_tcp_sockets/client/main/idf_component.yml

```diff
@@ -0,0 +1,9 @@
+## IDF Component Manager Manifest File
+dependencies:
+  ## Required IDF version
+  idf:
+    version: ">=4.1.0"
```

Comment:
This doesn't seem to be right.

Reply:
See https://github.com/espressif/esp-eth-drivers/pull/19#discussion_r1486253668